portal/views/portal.py

```diff
@@ -542,9 +542,10 @@ def legal():
 def terms_and_conditions():
     """ terms-and-conditions of use page"""
     gil = current_app.config.get('GIL')
-    response_text = VersionedResource.fetch_elements(app_text(Terms_ATMA.name_key()))[0]
+    content, _ = VersionedResource.fetch_elements(
+            app_text(Terms_ATMA.name_key()))
```

Comment:
This is still relying on Terms_ATMA for ePROMs - that expands to "Terms and Conditions URL", which I only see in the TrueNTH persistence.  I think we're missing that app_text from [ePROMs persistence](https://github.com/uwcirg/ePROMs-site-config/blob/develop/site_persistence_file.json)